server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/redirect/NotifyMailetsMessageTest.java

```diff
@@ -211,4 +212,94 @@ public void generateMessageShouldAddSizeWhenMimeMessageAsSome() throws Exception
                 "  MAIL FROM: null\n" +
                 "  Size (in bytes): 6\n");
     }
+
+    @Test
+    public void generateMessageShouldDecodeEncodedSubject() throws Exception {
+        String content = "MIME-Version: 1.0\r\n" +
+            "Subject: =?UTF-8?Q?Cl=c3=b4ture_&_Paie_du_mois?=\r\n" +
+            "Content-Type: text/plain; charset=utf-8\r\n" +
+            "\r\n" +
+            "test\r\n";
+
+        MimeMessage message = new MimeMessage(Session.getDefaultInstance(new Properties()), new ByteArrayInputStream(content.getBytes()));
+        FakeMail mail = FakeMail.from(message);
+
+        String generateMessage = new NotifyMailetsMessage().generateMessage("my message", mail);
+
+        assertThat(generateMessage).isEqualTo("my message\n" +
```

Comment:
you should just assert the subject (based on the test title)

Reply:
done